src/server/index.ts

```diff
@@ -85,9 +90,9 @@ async function main(args: any) {
         password: args.password,
       };
       makeSocketServer(server, options);
-      server.listen(wsPort, 'localhost', (err: Error) => {
```

Comment:
I believe removing this causes it to default to 0.0.0.0 - I would prefer to have the safer default of localhost!

Reply:
Oops good catch, this was a remnant left over from trying to make the dev mode support listening on arbitrary IPs which I decided against and reverted in the end.